### DISCUSSION TITLE
[A-L] (20 Giugno 2024) Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#416] [Esami A-L]
La seguente formula proposizionale in CNF è soddisfacibile? [5 punti]

$$
    \Large \\{\ \\{p,\ \neg q\\},\ \\{q,\ \neg s\\},\ \\{\neg p,\ \neg q,\ \neg s\\},\ \\{s\\}\ \\}.
$$

Se si risponde "SI" definire un assegnamento che la soddisfa, se si risponde "NO" dimostrare l’insoddisfacibilità usando la regola di Risoluzione.

**Comment by cek753** [comment]
<img width="519" height="280" alt="image" src="https://github.com/user-attachments/assets/27eedd1f-4f63-41d6-8a18-892d4e32251c" />
